### PR TITLE
new api for airdrop

### PR DESCRIPTION
**Pull request by jellyfish-tom**
## What does this PR do?
Introduces new apiClient for airdrop

## Todos:

- [X] Checked on iOS
- [ ] Checked on Android

### CODE REVIEW

File: src/config/index.ts [hunk 1]
```diff
@@ -15,4 +15,5 @@ export default {
   explorerUrl: Config.EXPLORER_URL,
   sentryDsn: Config.SENTRY_DSN,
   emailNotificationsApi: Config.EMAIL_NOTIFICATIONS_API,
+  airdropApi: Config.AIRDROP_API,
```
Comment (marek-siemieniuk-morawski) [inline src/config/index.ts:18]: The fact it's undefined doesn't break anything? I'd add to every `.env` file a record:
```
AIRDROP_API = ""
``` 
to be sure it's not an undefined

**Review by marek-siemieniuk-morawski — APPROVED** [review]

File: src/config/index.ts (hunk 1 above)
Comment (jellyfish-tom) [reply src/config/index.ts:18]: done

File: .env.beta.mainnet [hunk 2]
```diff
@@ -15,3 +15,4 @@ EXPLORER_URL=https://explorer.bitcoinvault.global
 SENTRY_DSN=https://dc67fd6d5c2949f2a6853e60eb69d899@o429172.ingest.sentry.io/5375289
 
 EMAIL_NOTIFICATIONS_API=https://btcv-notifcations-email.rnd.land/api
+AIRDROP_API = ""
```
Comment (marek-siemieniuk-morawski) [inline .env.beta.mainnet:18]: I know I said `AIRDROP_API = ""` but to keep consistency let's have it without whitespaces between `=` 😀

Comment (jellyfish-tom) [reply .env.beta.mainnet:18]: 🤣

**Pull request merged**